apps/cms/scripts/seed.mjs

```diff
@@ -128,179 +90,233 @@ async function main(args) {
-  await updateEntityRecord(
-    'homepage',
-    '',
-    {
-      featured_pack: packTemplates[packTemplates.length - 1].id,
-    },
-    token
+  console.log(
+    'This operation will overwrite any records in your database. Are you sure you want to proceed?'
   )
+  if ((await readlineAsync('> y/N: ')) !== 'y') {
+    console.log('Operation canceled.')
+    process.exit(0)
+  }
+  
+  const createBrands = await readlineAsync('> Create brands? (y/N): ')
 
-  /**
-   * Create collections.
-   */
-  const collectionFactories = await Promise.all(
-    Factory.buildList('collection', numCollections).map(async (item) => {
-      const color = randColor()
-
-      const collectionImage = await makeImage({
-        token,
-        text: item.translations[0].name,
-        color: '#000000',
-        backgroundColor: color,
-      })
-
-      const rewardImage = await makeImage({
-        width: 700,
-        height: 300,
-        token,
-        text: `Reward for ${item.translations[0].name}`,
-        font: 'bold 24px Arial',
-        color: '#ffffff',
-        backgroundColor: color,
-        lineWidth: 8,
-      })
-
-      item.collection_image = collectionImage.id
-      item.reward_image = rewardImage.id
-
-      return item
-    })
-  )
-  const collections = await createEntityRecords(
-    'collections',
-    collectionFactories,
-    token
-  )
+  await knex.raw(`TRUNCATE TABLE 
+    directus_files,
+    homepage,
+    brands, brands_translations,
+    rarities, rarities_translations, 
+    nft_templates, nft_templates_translations,
+    pack_templates, pack_templates_translations, pack_templates_directus_files,
+    collections, collections_translations,
+    sets, sets_translations
+    CASCADE`)
 
-  /**
-   * Create sets.
-   */
-  // Determine the number of of collectibles per collection based on input params.
-  // For example, if we have 48 collectibles and we want 3 collections, numCollectiblesPerCollection = 16.
-  const numCollectiblesPerCollection = numCollectibles / numCollections
-  // Chunk the collectibles evenly into collections.
-  const chunkedCollectionsOfCollectibles = chunkArray(
-    collectibles,
-    numCollectiblesPerCollection
-  )
-  await Promise.all(
-    collections.map(async (collection, i) => {
-      // For each collection, break the number of collectibles per collection evenly into sets of collectibles.
-      const chunkedSetsOfCollectibles = chunkArray(
-        chunkedCollectionsOfCollectibles[i],
-        numCollectiblesPerSet
-      )
-      // Finally, build the set factories and create the records.
-      const setFactories = chunkedSetsOfCollectibles.map((chunk) =>
-        Factory.build('set', {
-          collection: collection.id,
-          nft_templates: chunk.map((item) => item.id),
-        })
-      )
-      return await createEntityRecords('sets', setFactories, token)
-    })
-  )
+  console.time('Seed database')
+  console.log('createBrands', createBrands)
+  if (createBrands === 'y') {
```

Comment:
The diffs of this file are hot garbage. The essence of the changes are here. `seed` does what `main` did before but takes an optional `brandId` to tie the data to. The other changes are to ask whether or not to create brands and optionally creating them.